Tests/LibWeb/Text/input/HTML/storage-does-not-have-legacy-override-builtins-flag.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<script src="../include.js"></script>
+<script>
+    test(() => {
+        for (const functionName of ["key", "getItem", "setItem", "removeItem", "clear"]) {
+            const failFunction = function () { println(`FAIL: Overriden ${functionName} was called`); };
+            const failFunctionString = failFunction.toString();
```

Comment:
What is this variable used for?

Reply:
Good catch, I think I was just trying something out and forgot to remove it :^)